src/197F40.c

```diff
@@ -2277,7 +2277,23 @@ ApiStatus GetDistanceToGoal(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "197F40", func_8026EA7C);
+ApiStatus func_8026EA7C(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```

Comment:
The `args` you get out of `script->ptrReadPos` are always of the type `Bytecode`
```suggestion
    Bytecode* args = script->ptrReadPos;
```